### PR TITLE
chore(test): debug command

### LINKED ISSUE #8953: Fix k8s e2e tests that are failing on cicd
### Bug description

Currently all e2e tests that check k8s functionalities are broken on cicd and need to be fixed.

### Operating system

Windows 11

### Installation Method

Installer from website/GitHub releases

### Version

next (development version)

### Steps to reproduce

_No response_

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cbr7**
### What does this PR do?
Trying to fix k8s e2e tests that are failing

### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop/issues/8953

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "test:e2e:extension": "npm run test:e2e:build && npm run test:e2e:extension:run",
     "test:e2e:extension:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- npx playwright test tests/playwright/src/specs/extension-installation.spec.ts",
     "test:e2e:pw": "npm run test:e2e:build && npm run test:e2e:pw:run",
-    "test:e2e:pw:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- npx playwright test tests/playwright/src/specs/podman-machine-tests.spec.ts",
+    "test:e2e:pw:run": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' -- npx playwright test tests/playwright/src/specs/kind.spec.ts",
```
Comment (ScrewTSW) [inline package.json:47]: what is the intended purpose of this target?
the `pw` is very non-descriptive for me, and you completely changed the test it executed originally

File: tests/playwright/src/model/pages/cli-tools-page.ts [hunk 2]
```diff
@@ -34,5 +37,54 @@ export class CLIToolsPage extends SettingsPage {
...
+    return this.getToolRow(toolName).getByLabel('Update', { exact: true });
+  }
+
+  public getVersionSelectionButton(version: string): Locator {
+    return this.dropDownDialog.getByRole('button', { name: version });
+  }
+
+  public async getCurrentToolVersion(toolName: string): Promise<string> {
+    if ((await this.getToolRow(toolName).getByLabel('no-cli-version', { exact: true }).count()) > 0) {
+      return '';
```
Comment (ScrewTSW) [inline tests/playwright/src/model/pages/cli-tools-page.ts:68]: Is empty string safe?

File: tests/playwright/src/specs/kind.spec.ts [hunk 3]
```diff
@@ -35,8 +35,8 @@ const EXTENSION_LABEL: string = 'podman-desktop.kind';
 const CLUSTER_NAME: string = 'kind-cluster';
 const KIND_CONTAINER_NAME: string = `${CLUSTER_NAME}-control-plane`;
 const CLUSTER_CREATION_TIMEOUT: number = 200000;
-
 let resourcesPage: ResourcesPage;
+
```
Comment (ScrewTSW) [inline tests/playwright/src/specs/kind.spec.ts:39]: Why did this new line move? :D

**Review by ScrewTSW — APPROVED** [review]
Some comments and questions, but LGTM

File: package.json (hunk 1 above)
Comment (cbr7) [reply package.json:47]: this is just for debugging, the original test was also added by me

File: tests/playwright/src/model/pages/cli-tools-page.ts (hunk 2 above)
Comment (cbr7) [reply tests/playwright/src/model/pages/cli-tools-page.ts:68]: certainly, it permits validating against tobeTruthy() and toBeFalsy()

File: tests/playwright/src/specs/kind.spec.ts (hunk 3 above)
Comment (cbr7) [reply tests/playwright/src/specs/kind.spec.ts:39]: I fiddled around with the variable at some point and I guess the formatter liked it like this better when I added it back.

**Comment by cbr7** [comment]
e2e test run with this fix here: https://github.com/containers/podman-desktop-e2e/actions/runs/10924109666

**Pull request merged**